bedrock/mozorg/redirects.py

```diff
@@ -617,4 +617,7 @@ def decider(request, **kwargs):
     redirect(r'^about/patents/license/1.0/?$', 'mozorg.about.policy.patents.license-1.0'),
 
     redirect(r'^projects/marketing(/.*)?$', 'https://wiki.mozilla.org/MarketingGuide'),
+
+    # bug 1288647
+    redirect(r'^hacking/?$', 'https://developer.mozilla.org/en-US/docs/Hacking_Firefox'),
```

Comment:
There's not much on the redirected page, other than a simple link to `https://developer.mozilla.org/docs/Mozilla/Developer_guide/Introduction`. I'd suggest we redirect people there instead (note I left out `/en-US/` from the URL, since this means users will get directed to the most appropriate translation of the page).